src/development/vortex/development/utils/data/dataset/darknet.py

```diff
@@ -0,0 +1,122 @@
+from pathlib import Path
+from collections import namedtuple
+from typing import Dict, List, Union, Callable, Tuple
+from multipledispatch import dispatch
+
+import cv2
+import random
+import numpy as np
+import pandas as pd
+
+_file_path = Path(__file__)
+_repo_root = _file_path.parent.parent
+
+__all__ = [
+    'DarknetDataset',
+]
+
+supported_dataset = [
+    'DarknetDataset'
+]
+
+class DarknetDataset :
+    def __init__(self,
+                 txt_path: str, img_root: str, names: str,
+                 check_exists=True):
```

Comment:
why not make `check_exist` a default and drop the additional argument

Reply:
do you mean make this dataset strictly checking existence?

Reply:
yes, but probably there is a use case for that though as I realized just now. So I think it's all good

Reply:
ok